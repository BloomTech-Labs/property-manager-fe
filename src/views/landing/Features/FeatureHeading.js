import React from 'react';
import FeatureImage from '../../../assets/svg/features-image.svg';

const FeatureHeading = () => {
  return (
    <div className="featureHeadingWrapper">
      <img
        className="featureImage"
        src={FeatureImage}
        alt="Orange computer and phone"
      />
      <h1 className="featureHeading">Which Freehold is right for me?</h1>
    </div>
  );
};

export default FeatureHeading;
