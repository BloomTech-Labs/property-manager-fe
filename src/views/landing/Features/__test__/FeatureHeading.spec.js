import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom';
import FeatureHeading from '../FeatureHeading';

afterEach(rtl.cleanup);

test('Makes sure the word Freehold exists on LP', async () => {
  const wrapper = rtl.render(<FeatureHeading />);
  expect(await wrapper.queryAllByText(/freehold/i));
});

test('snapshot', () => {
  const { container } = rtl.render(<FeatureHeading />);
  expect(container.firstChild).toMatchSnapshot();
});
