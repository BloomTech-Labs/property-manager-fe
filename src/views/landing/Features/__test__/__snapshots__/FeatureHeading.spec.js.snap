// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  class="featureHeadingWrapper"
>
  <img
    alt="Orange computer and phone"
    class="featureImage"
    src="features-image.svg"
  />
  <h1
    class="featureHeading"
  >
    Which Freehold is right for me?
  </h1>
</div>
`;
